python/dgl/data/reddit.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import absolute_import
+
+import scipy.sparse as sp
+import numpy as np
+import dgl
+import os, sys
+from .utils import download, get_download_dir, _get_dgl_url
+
+
+class RedditDataset(object):
+    def __init__(self):
+        download_dir = get_download_dir()
+        # TODO upload the .npz files to S3
+        # graph
+        coo_adj = sp.load_npz(os.path.join(download_dir, "reddit_graph.npz"))
+        self.graph = dgl.DGLGraph(coo_adj, readonly=True)
+        # features and labels
+        reddit_data = np.load(os.path.join(download_dir, "reddit_data.npz"))
```

Comment:
do you reorganize your data? it's different from the ones I get from your script.